<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fafafa;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_14">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Image 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>             FT Component</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGraphicsView" name="label_img_1">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGraphicsView" name="FT_label_1">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="FT_img1">
             <property name="styleSheet">
              <string notr="true">background-color: #f0f2f6;</string>
             </property>
             <item>
              <property name="text">
               <string>Select A Component...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Magnitude</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Phase</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Real</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imaginary</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Image 4</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QGraphicsView" name="FT_label_4">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>             FT Component</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGraphicsView" name="label_img_4">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="FT_img4">
             <property name="styleSheet">
              <string notr="true">background-color: #f0f2f6;</string>
             </property>
             <item>
              <property name="text">
               <string>Select A Component...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Magnitude</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Phase</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Real</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imaginary</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Image 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="1">
            <widget class="QGraphicsView" name="FT_label_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGraphicsView" name="label_img_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>             FT Component</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="FT_img2">
             <property name="styleSheet">
              <string notr="true">background-color: #f0f2f6;</string>
             </property>
             <item>
              <property name="text">
               <string>Select A Component...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Magnitude</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Phase</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Real</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imaginary</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Image 3</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QGraphicsView" name="label_img_3">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>             FT Component</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGraphicsView" name="FT_label_3">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="FT_img3">
             <property name="styleSheet">
              <string notr="true">background-color: #f0f2f6;</string>
             </property>
             <item>
              <property name="text">
               <string>Select A Component...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Magnitude</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Phase</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Real</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imaginary</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>1000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f0f2f6;</string>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="3" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="reset_region_button">
           <property name="text">
            <string>Reset Regions</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="rectangle_region_button">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>6000</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #fafafa;</string>
           </property>
           <property name="text">
            <string>Select Regions</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>6000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Mixer Output to</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="choose_output">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>6000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #fafafa;</string>
             </property>
             <item>
              <property name="text">
               <string>Output 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Output 2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="toggle_shading_button">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Toggle Shading</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="Image2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Image_2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_9">
           <item row="0" column="0">
            <widget class="QSlider" name="ft_img_slider2">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Image_weight2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="9" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="Image4">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Image_4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QSlider" name="ft_img_slider4">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Image_weight4">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="Image3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Image_3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QSlider" name="ft_img_slider3">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Image_weight3">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="Image1">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Image_1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QSlider" name="ft_img_slider1">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Image_weight1">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Output 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QGraphicsView" name="graphicsView_10"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Output 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QGraphicsView" name="graphicsView_9"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1353</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
